src/main/vct/main/stages/Transformation.scala

```diff
@@ -210,6 +210,7 @@ case class SilverTransformation
     CollectLocalDeclarations, // all decls in Scope
     DesugarPermissionOperators, // no PointsTo, \pointer, etc.
     ReadToValue, // resolve wildcard into fractional permission
+    VariableToPointer,
```

Comment:
I think it would be best to keep this after `ParBlockEncoder` (which it is), because of this: https://github.com/utwente-fmt/vercors/issues/1165 . Maybe add a comment saying this.